automobile/scenario-creation-tutorial.md

```diff
@@ -68,7 +68,38 @@ To do this, open the `sumo.net.xml` file in SUMO `netedit`:
 $WEBOTS_HOME/projects/automobile/resources/sumo/bin/netedit $WBT_PROJECT_PATH/worlds/myMap_net/sumo.net.xml
 ```
 
-You can then generate the route file, SUMO provides several ways to generate route files. You may for example generate a [flow file](http://sumo.dlr.de/wiki/Definition_of_Vehicles,_Vehicle_Types,_and_Routes) and then use [duarouter](http://sumo.dlr.de/wiki/DUAROUTER) to generate the route file for you:
+
+### Generate the SUMO route file
+
+At this stage, a SUMO route file should be created to describe the traffic, i.e.,
+the number of vehicles, which road they should take, the vehicle properties, etc.
+
+This [SUMO document](http://sumo.dlr.de/wiki/Definition_of_Vehicles,_Vehicle_Types,_and_Routes) explains precisely how to achieve this.
+
+This following subsections describes two ways to create a SUMO route file.
+The use of the SUMO [duarouter](http://sumo.dlr.de/wiki/DUAROUTER) program is required in both cases.
+
+
+#### Create randomly a SUMO route file
+
+The purpose of the SUMO `randomTrips.py` script is to generate a random SUMO trip file.
+This file can be used as an input argument of `SUMO duarouter` to create the expected SUMO route file.
+Here are the commands to do so:
+
+```sh
+python $WEBOTS_HOME/projects/automobile/resources/sumo/tools/randomTrips.py -n $WBT_PROJECT_PATH/worlds/myMap_net/sumo.net.xml -o $WBT_PROJECT_PATH/worlds/myMap_net/sumo.trip.xml -e 3600 -p 2
```

Comment:
No problems. But honestly, I think the best is to simply drop these arguments.

Indeed, according to http://sumo.dlr.de/wiki/Tools/Trip , `-e 3600` is already the default value. And `-p 1` is the default value, but it won't change the result a lot. All the SUMO scripts are full of parameters, if a user want to change them, he needs to read the SUMO documentation.